nautobot/extras/utils.py

```diff
@@ -165,12 +165,11 @@ def __call__(self):
 
         return query
 
-    @property
```

Comment:
To make `TaggableClassesQuery.as queryset` consistent with both `ChangeLoggedModelsQuery.as queryset` and `FeatureQuery.as queryset`, `@property` was removed.
`as_queryset` is required to be consistent here, therefore it has to be done.
```python
queryset_map = {"tags": TaggableClassesQuery, "job hooks": ChangeLoggedModelsQuery}
form_field = MultipleContentTypeField( choices_as_strings=True, queryset=queryset_map[plural_name]().as_queryset() )
```